indico/modules/events/templates/display/administrative/event.html

```diff
@@ -0,0 +1,100 @@
+{% from 'events/display/indico/_common.html' import render_datetime %}
+{% from 'events/display/common/_manage_button.html' import render_manage_button %}
+{% from 'events/timetable/display/administrative/_common.html' import render_location %}
+
+{% macro render_event_time(event, timezone) %}
+    {% if event.start_dt.astimezone(timezone).date() == event.end_dt.astimezone(timezone).date() %}
+        {{ event.start_dt | format_date(format='full', timezone=timezone|string) }}
+        - <em class="time">{{ event.start_dt | format_time(format='medium', timezone=timezone) }}</em>
+    {% else %}
+        {% trans start_date=event.start_dt|format_date(format='full', timezone=timezone|string),
+                 start_time=event.start_dt|format_time(format='medium', timezone=timezone),
```

Comment:
you `|string` the timezone for `format_date` but not `format_time`. if the functions are really that broken maybe it'd be better to fix them so they are both (not) broken and accept both a timezone string and a tzinfo (assuming that's what you want to convert from with the `|string`)